### ISSUE TITLE
Scrobbles Do Not Upload In Background

### ISSUE THREAD

**Issue by ibrokemypie** [simple-last-fm-scrobbler/sls#357]
Scrobbles are getting recorded by the app fine but are not being submitted until I open the app, regardless of settings. SLS is set to submit on battery on any network and when track finishes.

**Comment by inverse** [comment]
Are you running any "battery saver apps"?

**Comment by ibrokemypie** [comment]
Only the normal doze, which i would have thought would effect the recording of played songs too if it were the problem...

**Comment by HumbleBeeBumbleBee** [comment]
What music player are you using?

**Comment by ibrokemypie** [comment]
Phonograph

**Comment by inverse** [comment]
Could you try and tell the system not to battery optimise SLS and see if the issue persists? If it goes away then we may need to look into how to allow scrobbling with doze enabled.

**Comment by Nicd** [comment]
I also seem to have this problem on Android 7.0, using Google Play Music. How do I disable battery optimisation for SLS? I can't find any such option under Settings -> Apps.

**Comment by hansdg1** [comment]
Ok, I checked both OnGoing and Phone Notification. I am now consistently
getting my scrobbles tracked!

Now, whenever I start listening to music I get a persistent notification
for SLS. This is fine except that it seems to stay there forever, even
after I stop listening to music. The only way I've been able to close it is
by killing the app. I understand the need for the notification, but I would
like for it to hide itself when I am no longer listening to music.

Thank you,

Hans
